src/steps/2.addModules/moduleConfigs.ts

```diff
@@ -309,7 +327,11 @@ declare interface ModuleConfig {
   nuxtConfig: NuxtConfig
   files: File[]
   tasksPostInstall: string[]
-  htmlForIndexVue?: string
+  indexVue?: {
+    html?: string,
+    css?: string
+    js?: string,
```

Comment:
are all of these truely optional? I think we can require `html` at least, right?